readme.md

```diff
@@ -76,6 +76,8 @@
   - [Mehdi Sakout](https://mehdisakout.com) - Mobile, Android, Indie Making, SaaS ...
   - [Ismail_Elazizi](https://ismailelazizi.com/blog) - Design, UX/UI, Design tips ...
   - [KAOUSSI Souhail](https://medium.com/@artisoft) - Javascript - Vue ...
+  - [Anas Mazouni](https://blog.anasmazouni.dev/) - PHP (Laravel), Javascript/Typescript, Python, C++, AI, and other random stuff ...
```

Comment:
Can you make teh description a bit briefer ? 2/3 topics :)

Reply:
Sure!